dev_docs/RELEASES.md

```diff
@@ -17,9 +17,9 @@ For patch releases, only the version on the existing major and minor version bra
   * Review existing [changelogs/head](https://github.com/elastic/apm-server/tree/main/changelogs/head.asciidoc) to ensure all relevant notes have been added.
```

Comment:
Question: how do I make sure there are no PR to be merged? I checked all PRs that don't have a `backport-skip` label applied through [this search](https://github.com/elastic/apm-server/pulls?q=is%3Apr+sort%3Acreated-desc+-label%3Abackport-skip). If that is appropriate I would add it to the documentation to help streamline this step.